packages/manager/apps/dedicated/client/app/billing/autoRenew/autorenew.html

```diff
@@ -215,8 +215,8 @@
                 <span
                     class="oui-badge"
                     data-ng-class="{
-                        'oui-status_success': !$row.isSuspended(),
-                        'oui-status_error': $row.isSuspended()
+                        'oui-badge_success': !$row.isSuspended(),
```

Comment:
Why replace with a `oui-badge_success` here but not above https://github.com/ovh/manager/pull/3190/commits/dba28c23efd1badd8a8acee7e85a0a8190e663f0#diff-24d0ffe21df10825c29e319593ed4e99R121
?

Reply:
Related to my others comments, I'll double check that. `oui-status` should have only migrate to `oui-badge`.